demo/src/store/store.js

```diff
@@ -0,0 +1,95 @@
+import {createStore} from 'redux';
+import airplaneEars from '../images/airplane_ears.jpg';
+import bigYawn from '../images/big_yawn.jpg';
+import fluffy from '../images/fluffy.jpg';
+import makinBiscuits from '../images/makin_biscuits.jpg';
+import pawsUp from '../images/paws_up.jpg';
+import petMyBelly from '../images/pet_my_belly.jpg';
+import phoneBackground from '../images/phone_background.jpg';
+import snoozing from '../images/snoozing.jpg';
+import spoiled from '../images/spoiled.jpg';
+import whatYouLookinAt from '../images/what_you_lookin_at.jpg';
+
+/**
+ * Functions to update the value saved in the store.
```

Comment:
Per 
https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

Method descriptions begin with a verb phrase that describes what the method does. This phrase is not an imperative sentence, but instead is written in the third person, as if there is an implied This method ... before it.

Reply:
Reworded